manager/apis/app/v1beta1/fybrikapplication_webhook.go

```diff
@@ -0,0 +1,71 @@
+// Copyright 2020 IBM Corp.
+// SPDX-License-Identifier: Apache-2.0
+
+package v1beta1
+
```

Comment:
The parallel file in v1alpha1 is not needed (main uses only v1beta1 webhook for fybrikapplications)

Reply:
No we need webhooks for both versions and main uses all of them.

Reply:
Right, I did not see the usage.